api_tests/institutions/views/test_institution_summary_metrics.py

```diff
@@ -42,10 +42,6 @@ def test_get(self, app, url, institution, user, admin):
         resp = app.get(url, auth=user.auth, expect_errors=True)
         assert resp.status_code == 403
 
-        # Tests authorized user with institution without any metrics
```

Comment:
This was just a bad test I think.

Reply:
The response for an institution that exists but does not have any metrics data should return a 404 in my mind. The test was failing because of the removal of this bit (https://github.com/CenterForOpenScience/osf.io/pull/9355/files#diff-0e5a84f6966252c2a797725de92cd9c3L402). The response for an institution the exists but does not have any metrics data under this PR looks like:
```
{
    "meta": {
        "version": "2.0"
    }
}
```

Reply:
Yeah I don't think a 404 is appropriate here, ussually that's reserved for situations when you miss the endpoint totally.